config/storybook/preview-head.html

```diff
@@ -1,4 +1,8 @@
 <script>
-  window.__ttn_config__ = {}
+  window.__ttn_config__ = {
+    APP_CONFIG: {
+      documentation_base_url: 'https://thethingsstack.io',
+    },
+  }
```

Comment:
Let's add the other configs as well. We can just copy it out from the real deployment and adapt it.

Reply:
Why? In general base components should rely on global env variables and we shouldnt encourage such functionality for base component. Usage of env selectors in `<Link.DocLink />` is questionable, I would prefer passing those via the global context. It is as it is atm, but I dont think we should expose the whole env object to base components storybook.

Note, this was added only to fix storybook.